app.py

```diff
@@ -52,33 +51,36 @@ def run_app(self):
         (current_in, current_out) = self.get_current_frame_range(self.engine.name)
 
         if new_in is None or new_out is None:
-            message =  "Shotgun has not yet been populated with \n"
-            message += "in and out frame data for this Shot."
+            message = ("Shotgun has not yet been populated with \n"
+                       "in and out frame data for this Shot.")
```

Comment:
We are not really far from their code. ATM, the only thing that we added is the XSI support and we did one or two hotfix (like force variables to be int when setting frame range in nuke)